plantcv/plantcv/fill.py

```diff
@@ -2,13 +2,11 @@
 
 import numpy as np
 import os
-from plantcv.plantcv import print_image
-from plantcv.plantcv import plot_image
 from plantcv.plantcv import fatal_error
 from plantcv.plantcv import params
+from plantcv.plantcv._debug import _debug
 from skimage.morphology import remove_small_objects
 
-
```

Comment:
We follow the Python community style recommendations, which specifies two blank lines prior to a function.